zomes/holo_acorn/code/src/lib.rs

```diff
@@ -134,7 +134,9 @@ pub struct ArchiveGoalResponse {
 #[derive(Serialize, Deserialize, Debug, DefaultJson, Clone)]
 pub struct GetHistoryResponse {
     entries: Vec<Goal>,
+    members:Vec<Vec<GoalMember>>,
```

Comment:
Why is it `Vec<Vec<GoalMember>>`?
Why not `<Vec<GoalMember>>`

Reply:
because an array of the change history of each goal_member

Reply:
members[1][n]= represents the n changes of the first goal_members(create, delete and updates)

members[2][n]= represents the n changes of the second goal_member (create, delete and updates)

and so